uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -309,18 +350,25 @@ sub get_acquisitions {
 ################################################################
 sub computeMd5Hash {
     my $this = shift;
-    my ($file) = @_;
-    $this->{LOG}->print(
-        "==> computing md5 hash for MINC body.\n"
-    ) if $this->{verbose};
+    my ($file, $tarchive_id) = @_;
```

Comment:
I could only find one call to computeMd5Hash: in uploadNeuroDB/minc_insertion.pl. You updated the call made in that file so I would say everything is OK with respect to this API change.